code/modules/admin/quantum_mechanic.dm

```diff
@@ -0,0 +1,323 @@
+/*
+// Quantum Mechanic is a godmode avatar designed for debugging and admin actions
+// Their primary benefit is the ability to spawn in wherever you are, making it quick to get a human for your needs
+// They also have incorporeal flying movement if they choose, which is often the fastest way to get somewhere specific
+// They are mostly invincible, although godmode is a bit imperfect.
+// Most of their superhuman qualities can be toggled off if you need a normal human for testing biological functions
+*/
+
+/client/proc/quantum_mechanic()
+	set category = "Debug"
+	set name = "Spawn Quantum Mechanic"
+	set desc = "Spawns a Quantum Mechanic to debug stuff."
+
+	if(!check_rights(R_ADMIN|R_DEBUG))
+		return
+
+	if(!isghost(mob))
+		to_chat(src, SPAN_WARNING("You must to be a ghost to use this!"))
+		return
+
+	var/T = get_turf(mob)
+	var/mob/living/carbon/human/quantum/Q = new(T)
+
+	Q.dir = mob.dir
+	Q.ckey = ckey
+	Q.name = "Quantum Mechanic"
+	Q.real_name = "Quantum Mechanic"
```

Comment:
Why not put all these vars on `/mob/living/carbon/human/quantum` instead of setting them at runtime?